### PR TITLE
Use the third person in method docs

### PR DESCRIPTION
**Pull request by r00ster91**
There are some methods left that don't use the third person.

### CODE REVIEW

**Comment by vladfaust** [comment]
In my projects I always use no-person references in the method introduction and then the third person in description. E.g:

```crystal
# Set foo to bar. May raise. Uploads to server.
def foo
end
```

So in the docs summary it says `Set foo to bar` and in the full description there is a third person reference to the described method.

Which approach is right?

**Comment by r00ster91** [comment]
According to [Documenting code](https://crystal-lang.org/docs/conventions/documenting_code.html) you should always document methods in the third person:
> - Use the third person: `Returns the number of horns this unicorn has` instead of `Return the number of horns this unicorn has`.

File: src/array.cr [hunk 1]
```diff
@@ -190,7 +190,7 @@ class Array(T)
     size <=> other.size
   end
 
-  # Set intersection: returns a new `Array` containing elements common to `self`
+  # Sets intersection: returns a new `Array` containing elements common to `self`
```
Comment (z64) [inline src/array.cr]: This is actually a noun, not a verb. ("Set intersection", as in the intersection of two sets)

Same below on the next diff line for this file

File: src/compiler/crystal/compiler.cr [hunk 2]
```diff
@@ -83,7 +83,7 @@ module Crystal
     # one LLVM module is created for each type in a program.
     property? single_module = false
 
-    # Set to a `ProgressTracker` object which tracks compilation progress.
+    # Sets to a `ProgressTracker` object which tracks compilation progress.
```
Comment (RX14) [inline src/compiler/crystal/compiler.cr]: This was correct before, although badly worded (my docs, sorry)

File: src/compiler/crystal/program.cr [hunk 3]
```diff
@@ -110,7 +110,7 @@ module Crystal
     # The main filename of this program
     property filename : String?
 
-    # Set to a `ProgressTracker` object which tracks compilation progress.
+    # Sets to a `ProgressTracker` object which tracks compilation progress.
```
Comment (RX14) [inline src/compiler/crystal/program.cr]: ditto

**Pull request merged**